hadoop-hdds/docs/content/feature/Decommission.md

```diff
@@ -78,14 +78,15 @@ Storage Container Manager (SCM) decommissioning is the process in which you can
 To decommission a SCM and remove the node from the SCM HA ring, the following steps need to be executed.
 ```shell
 ozone admin scm decommission [-hV] [-id=<scmServiceId>] -nodeid=<nodeId>
-                                    [--scm=<scm>]
 ```
 You can obtain the 'nodeId' by executing this command, **"ozone admin scm roles"**
 
-**Note -** If you want to decommission a **primordial** scm, you may first transfer the leadership to a non primordial scm and then decommission the primordial scm.
+**Note -** If you want to decommission the **leader** scm, you must first transfer the leadership to a different scm and then decommission the node.
 
 To transfer the leader, we can excute below command,
 ```shell
-ozone admin scm transfer --service-id=scmservice -n ${nodeId}
+ozone admin scm transfer [-id=<scmServiceId>] -n=<nodeId>
```

Comment:
Please do not change `--service-id` to `-id`.  Both are supported by the command, but `-id`, as a multi-char short option, should really be deprecated (see description of HDDS-6736).